### ISSUE TITLE
Feedback: Awesome Extension! 

### ISSUE THREAD

**Issue by Foxxsayy** [JasonGrass/auto-extension-manager#140]
Just wanted to say I love your extension, use it all the time. The feature set is incredible and it comes in really handy. Thanks!

---

Extension Manager 0.5.19
Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36

**Comment by neoOpus** [comment]
Yeah it is great and I just posted asking for few more features (https://github.com/JasonGrass/auto-extension-manager/issues/144) to make it even better

**Comment by JasonGrass** [comment]
THANKS
